org.eclipse.xtext.xbase/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Xbase Model
 Bundle-Vendor: Eclipse Xtext
-Bundle-Version: 2.18.0.qualifier
+Bundle-Version: 2.17.0.qualifier
```

Comment:
looks like this needs a rebase @epoell

Reply:
In that MANIFEST I was now able to change the version.
But somehow I must have missed something at the rebase and it might still be the wrong version for all the other manifests, despite I have version 2.18 in all files locally...